docs/style-guide.md

```diff
@@ -1,30 +1,30 @@
 # Code style guide
 
-The document introduces the style guide for codes submited to poi. Codes of good and coherent style is friendly to other contributors and yourself in the future. Please follow them.
+This document introduces the style guide for code submitted to poi. Good and coherent code style is friendly to other contributors and yourself in the future. Please follow them.
```

Comment:
Let's try this.   
This document is the style guide for poi. Good and coherent style is friendly to other contributors and yourself in the future. Please follow them.